### DISCUSSION TITLE
Contributing instructions

### DISCUSSION THREAD

**Discussion by Gabe-Vince** [Jonak-Adipta-Kalita/JAK-Mobile-App#711] [General]
Hey @Jonak-Adipta-Kalita I would like to contribute to the repo but when I go through the CONTRIBUTING.md there is and error.

I am unable to run build:android because this error shows up:

```
You must have access to the xxjonakadiptaxx account to configure this EAS project.

```

Let me know the steps forward

**Comment by Jonak-Adipta-Kalita** [comment]
I appreciate that you want to contribute to the project. But my Finals are going on at the moment and I am unable to maintain the repo for some time (1 month to be exact). I will be back after my Finals are over.

Anyway, for the above issue, it seems to me that you wouldn't be able to `build` the app since its configured to target my account. I will fix it when I am free. I will see you then :)

**Reply by Jonak-Adipta-Kalita** [reply]
And without building the app, I dont think you will be able to spin up the Expo Server (since the Firebase stuff are needed)
